### PR TITLE
Update manure metadata patterns

### LINKED ISSUE #1134: Adding and Deleting Manure Metadata patterns
## Expected Behavior
There are some missing patterns and some extra patterns in the manure management metadata. They need to be corrected to align with the options available for simulation

## Possible Solution

1. bedding_type - delete 'Other', add 'CBPB sawdust'
2. manure_handler - delete 'manual skid steer scraping'
3. "manure_separator" - delete 'other'
4. "manure_treatment_methods" - see comments in [this issue](https://github.com/RuminantFarmSystems/MASM/issues/1095)

Note that the patterns need to be changed in the "manure_management_scenarios" sections as well as the config sections. 

## Steps to Reproduce

1.
6.
7.
8.

### PR DESCRIPTION
**Pull request by ew3361zh**
PR to address issue #1134 to update metadata manure patterns in both the manure manager config sections and the manure management scenarios sections.

For each of the metadata files:
1. `bedding_type` has had "other" pattern removed and "CBPB sawdust" pattern added. 
2. `manure_handler` has had "manual skid steer scraping" pattern removed.
3. `manure_separator` has had "other" and "sand lane" patterns removed.
4. `manure_treatment_methods` now has the following patterns to match from: "^(slurry storage underfloor|slurry storage outdoor|open lots|compost bedded pack barn|anaerobic lagoon|anaerobic digestion and lagoon|anaerobic digestion and lagoon with separator)$"

### CODE REVIEW

**Review by KFosterReed — COMMENTED** [review]
I made a couple of small changes to the manure json inputs so they align with the metadata. Looks good! THank you Niko!

**Comment by ew3361zh** [comment]
> I made a couple of small changes to the manure json inputs so they align with the metadata. Looks good! THank you Niko!

Thanks, Kristan. After further investigation, the "CBPB sawdust" input should stay without the underscore as you had it originally. I will make that change right now.

**Comment by ew3361zh** [comment]
@KFosterReed could you approve the PR if you're good with the updates? Thank you!

**Review by KFosterReed — APPROVED** [review]

File: input/metadata/ARL_metadata.json [hunk 1]
```diff
@@ -3189,7 +3189,7 @@
             "manure_separator_type": {
               "type": "string",
               "description": "Manure Separator Type -- The type of solid-liquid separator equipment to separate coarse fibrous solids/sand",
-              "pattern": "^(screw press|sand lane|rotary screen|none|other)$"
+              "pattern": "^(screw press|sand lane|rotary screen|none)$"
             },
```
Comment (tomhuhh) [inline input/metadata/ARL_metadata.json:3193]: See in Julie's notes: “sand lane” and “other” to be deleted from metadata options.

So I think we should also delete 'sand lane' as an option for manure separator?

**Review by tomhuhh — COMMENTED** [review]
Hello @ew3361zh! This looks great to me. Only minor suggestions. Please see my comments in line.

File: input/metadata/ARL_metadata.json (hunk 1 above)
Comment (ew3361zh) [reply input/metadata/ARL_metadata.json:3193]: @KFosterReed Haowen made a good catch here. In Julie's notes (which I shared with Haowen for reference), she mentioned getting rid of both the "other" and "sand lane" options from manure separator. Can you confirm that's still the agreed-upon way forward?

Comment (tomhuhh) [reply input/metadata/ARL_metadata.json:3193]: Right now on the main branch, only two manure separation methods have configurations - 'rotary screen' and 'screw press'. So it makes sense to delete 'sand lane'. However, my question is do we have plans to add it as an option in the future?

Comment (KFosterReed) [reply input/metadata/ARL_metadata.json:3193]: Thanks for following up on this! Yes, let's remove sand lane as well.

Comment (ew3361zh) [reply input/metadata/ARL_metadata.json:3193]: Sand lane has been removed as an option. Thanks!

**Review by tomhuhh — APPROVED** [review]

**Pull request merged**